### PR TITLE
Implemented show logging buffered

### LINKED ISSUE #39: Create a stub function for show logging buffered
Work required for sprint 1 reference is #11

### PR DESCRIPTION
**Pull request by LondonWesley**
platform win32 -- Python 3.8.5, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: C:\Users\wesley\Desktop\softeng\comp410_fall_2020
plugins: cov-2.10.1
collected 8 items

asa_parser\tests\test_parser.py ........                                                                         [100%]

----------- coverage: platform win32, python 3.8.5-final-0 -----------
Name                              Stmts   Miss  Cover
-----------------------------------------------------
asa_parser\__init__.py                2      0   100%
asa_parser\src\asa_parse.py          34      7    79%
asa_parser\src\shtech_parse.py       27      1    96%
asa_parser\tests\__init__.py          0      0   100%
asa_parser\tests\test_parser.py      29      1    97%
-----------------------------------------------------
TOTAL                                92      9    90%

================================================== 8 passed in 1.50s ==================================================

(py38) C:\Users\wesley\Desktop\softeng\comp410_fall_2020>

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -43,5 +43,9 @@ def run_demo():
     # ipsec stats
     print(primary_asa.ipsec_stats())
 
+    # show logging buffered
+    print(primary_asa.show_logging_buffered())
```
Comment (claesmk) [inline demo.py:61]: This doesn't currently return anything due to the error noted in the next comment.
```
{"text": []}
```
Once you've corrected that problem the output should look something like this:
```
{"text": ["Aug 16 2017 15:35:37 KP-systest-admin : %ASA-4-711004: Task ran for 114 msec, Process = Unicorn Admin Handler, PC = f34bf8f4, Call stack = ", "Aug 16 2017 15:36:12 KP-systest-admin : %ASA-4-711004: Task ran for 108 msec, Process ...
```

File: asa_parser/src/asa_parse.py [hunk 2]
```diff
@@ -57,4 +57,8 @@ def show_cpu_detailed(self):
 
     def ipsec_stats(self):
         """Parser for show ipsec stats"""
-        return json.dumps({'stats': self.get_show_section('ipsec stats')})
\ No newline at end of file
+        return json.dumps({'stats': self.get_show_section('ipsec stats')})
+    
+    def show_logging_buffered(self):
+        """Parser for show show logging buffered"""
+        return json.dumps({'text': self.get_show_section('show logging buffered')})
```
Comment (claesmk) [inline asa_parser/src/asa_parse.py]: the get_show_section() function automatically handles the word "show" - actually it's designed to handle custom prefixes too
```
    def get_section(self, sec_heading='show', sec_name=None):
```

Please change this to 
```
return json.dumps({'text': self.get_show_section('logging buffered')}
```
and the rest should work OK

**Review by claesmk — CHANGES_REQUESTED** [review]

File: asa_parser/src/asa_parse.py (hunk 2 above)
Comment (claesmk) [reply asa_parser/src/asa_parse.py]: @LondonWesley any questions about the requested changes?

**Comment by claesmk** [comment]
I think you checked-in the hidden .vs directory by mistake - please remove these files

**Comment by LondonWesley** [comment]
Ok that should fix all the issues. Let me know if anything else arises.

**Review by claesmk — APPROVED** [review]

**Pull request merged**